content/shared/influxdb3-admin/query-system-data/_index.md

```diff
@@ -0,0 +1,141 @@
+
+_Note: when you send a query in JSON, you must escape single quotes
+that surround field names._
+
+```bash
+curl "http://localhost:8181/api/v3/query_sql" \
+  --header "Content-Type: application/json" \
+  --json '{
+    "db": "mydb",
+    "q": "SELECT * FROM information_schema.columns WHERE table_schema = '"'iox'"' AND table_name = '"'system_swap'"'",
+    "format": "jsonl"
+  }'
+```
+
+The output is the following:
+
+```jsonl
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"free","ordinal_position":0,"is_nullable":"YES","data_type":"UInt64"}
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"host","ordinal_position":1,"is_nullable":"NO","data_type":"Dictionary(Int32, Utf8)"}
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"percent","ordinal_position":2,"is_nullable":"YES","data_type":"Float64","numeric_precision":24,"numeric_precision_radix":2}
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"sin","ordinal_position":3,"is_nullable":"YES","data_type":"UInt64"}
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"sout","ordinal_position":4,"is_nullable":"YES","data_type":"UInt64"}
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"time","ordinal_position":5,"is_nullable":"NO","data_type":"Timestamp(Nanosecond, None)"}
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"total","ordinal_position":6,"is_nullable":"YES","data_type":"UInt64"}
+{"table_catalog":"public","table_schema":"iox","table_name":"system_swap","column_name":"used","ordinal_position":7,"is_nullable":"YES","data_type":"UInt64"}
+```
+
+#### Recently executed queries
+
+To view recently executed queries, query the `queries` system table:
+
+```bash
+curl "http://localhost:8181/api/v3/query_sql" \
+  --header "Content-Type: application/json" \
+  --json '{
+    "db": "mydb",
+    "q": "SELECT * FROM system.queries LIMIT 2",
+    "format": "jsonl"
+  }'
+```
+
+The output is the following:
+
+```jsonl
+{"id":"cdd63409-1822-4e65-8e3a-d274d553dbb3","phase":"success","issue_time":"2025-01-20T17:01:40.690067","query_type":"sql","query_text":"show tables","partitions":0,"parquet_files":0,"plan_duration":"PT0.032689S","permit_duration":"PT0.000202S","execute_duration":"PT0.000223S","end2end_duration":"PT0.033115S","compute_duration":"P0D","max_memory":0,"success":true,"running":false,"cancelled":false}
+{"id":"47f8d312-5e75-4db2-837a-6fcf94c09927","phase":"success","issue_time":"2025-01-20T17:02:32.627782","query_type":"sql","query_text":"show tables","partitions":0,"parquet_files":0,"plan_duration":"PT0.000583S","permit_duration":"PT0.000015S","execute_duration":"PT0.000063S","end2end_duration":"PT0.000662S","compute_duration":"P0D","max_memory":0,"success":true,"running":false,"cancelled":false}
+```
```

Comment:
Not needed for this PR, but we need to add the CLI instructions to this page too:

[influxdb3 show system](https://docs.influxdata.com/influxdb3/enterprise/reference/cli/influxdb3/show/system/)